src/material-experimental/mdc-menu/menu.ts

```diff
@@ -6,20 +6,52 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {ChangeDetectionStrategy, Component, ViewEncapsulation} from '@angular/core';
+import {ChangeDetectionStrategy, Component, ViewEncapsulation, Provider} from '@angular/core';
+import {Overlay, ScrollStrategy} from '@angular/cdk/overlay';
+import {
+  MatMenu as BaseMatMenu,
+  MAT_MENU_PANEL,
+  matMenuAnimations,
+  MAT_MENU_SCROLL_STRATEGY,
+} from '@angular/material/menu';
+
+/** @docs-private */
+export function MAT_MENU_SCROLL_STRATEGY_FACTORY(overlay: Overlay): () => ScrollStrategy {
+  return () => overlay.scrollStrategies.reposition();
+}
+
+/** @docs-private */
+export const MAT_MENU_SCROLL_STRATEGY_FACTORY_PROVIDER: Provider = {
+  provide: MAT_MENU_SCROLL_STRATEGY,
+  deps: [Overlay],
+  useFactory: MAT_MENU_SCROLL_STRATEGY_FACTORY,
+};
 
 @Component({
   moduleId: module.id,
   selector: 'mat-menu',
   templateUrl: 'menu.html',
   styleUrls: ['menu.css'],
-  host: {
-    'class': 'mat-mdc-menu',
-  },
-  exportAs: 'matMyCheckbox',
```

Comment:
lol, did I accidentally add this for all of the components in my skeleton PR? dangers of copy/paste...